mathewDavidov/assignments/twopointers/lc1/lc1.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Problem Statement: Given an array of sorted numbers and a target sum, find a
+ * pair in the array whose sum is equal to the given target. Write a function to
+ * return the indices of the two numbers (i.e. the pair) such that they add up
+ * to the given target.
+ * 
+ * Edge cases: 
+ * empty array / null array -> empty array 
+ * no target pair found -> empty array
+ * 
+ * Left pointer will begin at the first index (0) 
+ * Right pointer will begin at the last index (nums.length - 1) 
+ * while left < right 
+ *   create a currentSum equal to the numbers at the indices left and right 
+ *   if the currentSum is equal to the target, return the pair of indices in an array 
+ *   if the currentSum is less than the target, increment the left pointer 
+ *   if the currentSum is greater than the target, decrement the right pointer
+ * 
+ * Runtime:
+ * Time: O(n)
+ * Space: O(1)
```

Comment:
Love the psuedo-code. It was written well and in simple words. I really loved that you were able to write out the time and space complexity of your psuedo-code.